app/styles/app.scss

```diff
@@ -187,3 +187,12 @@ button .material-icons {
         }
     }
 }
+
+
+
+.sticky-nav{
+    width: 100%;
+    position: fixed !important;
+    top: 0;
```

Comment:
Is this taking into account the existence of edit bar? how is it made sure to not be over or under it?